### DISCUSSION TITLE
File URL Names with Spaces

### DISCUSSION THREAD

**Discussion by countstoneula** [iterate-ch/cyberduck#14429] [Q&A]
When I upload my files the ones with spaces in the filenames get a %20 in their URL names. I am trying to replace a bunch of old missing files on my server that already have links on my website. They all have a "-" in the links whenever there was a space. Is there a setting I can change so when I upload files with cyberduck it gives them a - instead of the %20 for spaces. This will just save me a ton of time needing to fix broken file links all over the place.

I've looked for answers online but have no idea what this file naming protocol is even called, and if it's a setting within Cyberduck or through my server.

**Comment by AliveDevil** [comment]
Translating dashes to spaces is probably a server-side configuration that Cyberduck is not aware of, and cannot support this scenario.
So copy the url, then replace-all `%20` with `-`. What that option might be called, or how it is configured, depends on the server software used.
